test/ExecutionFramework.cpp

```diff
@@ -282,3 +282,21 @@ bool ExecutionFramework::storageEmpty(h160 const& _addr) const
 	}
 	return true;
 }
+
+vector<solidity::frontend::test::LogRecord> ExecutionFramework::recordedLogs() const
+{
+	vector<solidity::frontend::test::LogRecord> logs{};
+	for (size_t logIdx = 0; logIdx < numLogs(); ++logIdx)
```

Comment:
Can't we use some kind of map feature from ranges_v3 instead of the iteration here? :)